src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockSpire/boss_pyroguard_emberseer.cpp

```diff
@@ -253,13 +217,13 @@ class boss_pyroguard_emberseer : public CreatureScript
                             break;
                         case EVENT_PLAYER_CHECK:
                         {
-                            // Check to see if all players in instance have aura SPELL_EMBERSEER_START before starting event
-                            bool _hasAura = true;
+                            // As of Patch 3.0.8 only one person needs to channel the altar
+                            bool _hasAura = false;
                             Map::PlayerList const& players = me->GetMap()->GetPlayers();
                             for (Map::PlayerList::const_iterator itr = players.begin(); itr != players.end(); ++itr)
                                 if (Player* player = itr->GetSource()->ToPlayer())
-                                    if (!player->HasAura(SPELL_EMBERSEER_OBJECT_VISUAL))
-                                        _hasAura = false;
+                                    if (player->HasAura(SPELL_EMBERSEER_OBJECT_VISUAL))
+                                        _hasAura = true;
```

Comment:
should there be a break; after this to stop at the first found player ? (with the required { } etc)

Reply:
Yes, you're absolutely right.